frontend/src/components/atoms/Shared/controls/Controls.tsx

```diff
@@ -0,0 +1,9 @@
+import Input from "./Input";
+import Button from "./Button";
+
+const Controls = {
```

Comment:
Oj, oj,oj. Zmiana Controls zawsze jak chcemy dodac cos nowego? Checbox / Radio button? Wszystko to mamy. To raczej tez nie dobra droga?

Reply:
oooł raaajt